src/client/apps/settings/client/curations/display/test/components/canvas/canvas_images.test.js

```diff
@@ -165,22 +176,27 @@ describe('Canvas Images', () => {
         )
         const input = component.find(ImageUpload).at(1).getElement()
         input.props.onChange(input.props.name, 'http://new-image.jpg')
-        const onChangeArgs = props.onChange.mock.calls[3]
+        const onChangeArgs = props.onChange.mock.calls[0]
 
         expect(onChangeArgs[0]).toMatch('canvas.assets')
         expect(onChangeArgs[1][0].url).toMatch('http://new-image.jpg')
         expect(onChangeArgs[2]).toBe(props.index)
       })
 
       it('Can remove an image', () => {
+        props.campaign.canvas.logo = 'http://artsy.net/logo.jpg'
+        props.campaign.canvas.assets = [
+          {url: 'http://artsy.net/image1.jpg'},
+          {url: 'http://artsy.net/image1.jpg'}
+        ]
         const component = mount(
           <CanvasImages {...props} />
         )
-        component.find('.RemoveButton').at(1).simulate('click')
-        const onChangeArgs = props.onChange.mock.calls[4]
+        component.find('.RemoveButton').at(2).simulate('click')
```

Comment:
I'm curious why you passed the class name versus the actual component similar to how you did in the header test file?

Reply:
For some reason the click wasn't working here in the same way, can try with `RemoveButtonContainer`.

Reply:
Ah gotcha. Yeah I don't think it's reason to hold up the PR but I know we are trying to transition to passing the actual component itself so I wanted to mention it. 

Maybe see if you can get it to work with `RemoveButtonContainer` and if not no biggy. Feel free to merge when you are ready.